server/data/appealClient.test.ts

```diff
@@ -0,0 +1,44 @@
+import AppealClient from './appealClient'
+import config from '../config'
+import { appealFactory, newAppealFactory } from '../testutils/factories'
+import describeClient from '../testutils/describeClient'
+import paths from '../paths/api'
+
+describeClient('AppeasClient', provider => {
+  let appealsClient: AppealClient
+
+  const token = 'token-1'
+
+  beforeEach(() => {
+    config.flags.oasysDisabled = false
```

Comment:
do we need this?

Reply:
I don't believe we do.. it doesn't seem to be used in the appealsClient, and if I remove it the tests pass